framework/console/controllers/MigrateController.php

```diff
@@ -202,12 +202,23 @@ protected function getMigrationHistory($limit)
         $query = new Query;
         $rows = $query->select(['version', 'apply_time'])
             ->from($this->migrationTable)
-            ->orderBy('apply_time DESC, version DESC')
             ->limit($limit)
             ->createCommand($this->db)
             ->queryAll();
-        $history = ArrayHelper::map($rows, 'version', 'apply_time');
-        unset($history[self::BASE_MIGRATION]);
+        $history = [];
+        foreach ($rows as $row) {
+            if ($row['version'] === self::BASE_MIGRATION) {
+                continue;
+            } elseif (preg_match('/m?(\d{6}_?\d{6})(\D.*)?$/is', $row['version'], $matches)) {
```

Comment:
`elseif` does not make sense. If previous condition is true, `continue` will go to next iteration of `foreach`